portal-impl/src/com/liferay/portal/service/persistence/impl/OrganizationFinderImpl.java

```diff
@@ -576,6 +591,94 @@ protected int countByOrganizationId(
 		return 0;
 	}
 
+	protected List<Object> doFindO_U_ByC_P(
+		long companyId, long parentOrganizationId,
+		QueryDefinition<?> queryDefinition) {
+
+		Session session = null;
+
+		try {
+			session = openSession();
+
+			StringBundler sb = new StringBundler(5);
+
+			sb.append(StringPool.OPEN_PARENTHESIS);
+
+			String sql = CustomSQLUtil.get(FIND_BY_C_P);
+
+			sql = StringUtil.replace(
+				sql, "SELECT organizationId",
+				"SELECT organizationId, 0 AS userId, name");
+			sql = StringUtil.replace(
+				sql, "(organizationId > ?) AND", StringPool.BLANK);
```

Comment:
The strings are in the query. They are just formatted differently with line breaks and spaces  https://github.com/liferay/liferay-portal/blob/master/portal-impl/src/custom-sql/portal.xml#L598-L610. 

I agree using string replacement in queries can be error-prone and confusing. I'll create a separate query for this so that it's more robust and clear.